[2025-05-07T05:00:03.945+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-07T05:00:03.969+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl_dag.run_dbt scheduled__2025-05-07T04:00:00+00:00 [queued]>
[2025-05-07T05:00:03.978+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl_dag.run_dbt scheduled__2025-05-07T04:00:00+00:00 [queued]>
[2025-05-07T05:00:03.979+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-05-07T05:00:03.990+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_dbt> on 2025-05-07 04:00:00+00:00
[2025-05-07T05:00:03.997+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1816) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-07T05:00:03.998+0000] {standard_task_runner.py:63} INFO - Started process 1818 to run task
[2025-05-07T05:00:03.997+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'weather_etl_dag', 'run_dbt', 'scheduled__2025-05-07T04:00:00+00:00', '--job-id', '165', '--raw', '--subdir', 'DAGS_FOLDER/weather_etl_dag.py', '--cfg-path', '/tmp/tmpfyy9msma']
[2025-05-07T05:00:04.000+0000] {standard_task_runner.py:91} INFO - Job 165: Subtask run_dbt
[2025-05-07T05:00:04.014+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-07T05:00:04.050+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_etl_dag.run_dbt scheduled__2025-05-07T04:00:00+00:00 [running]> on host 8676c2e751c4
[2025-05-07T05:00:04.130+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_etl_dag' AIRFLOW_CTX_TASK_ID='run_dbt' AIRFLOW_CTX_EXECUTION_DATE='2025-05-07T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-07T04:00:00+00:00'
[2025-05-07T05:00:04.132+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-07T05:00:10.973+0000] {logging_mixin.py:188} INFO - dbt run failed: Command '['dbt', 'run']' returned non-zero exit status 1.
[2025-05-07T05:00:10.974+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-07T05:00:10.975+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-07T05:00:10.984+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_etl_dag, task_id=run_dbt, run_id=scheduled__2025-05-07T04:00:00+00:00, execution_date=20250507T040000, start_date=20250507T050003, end_date=20250507T050010
[2025-05-07T05:00:11.019+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-07T05:00:11.034+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-07T05:00:11.037+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
